spaceship.zsh

```diff
@@ -1062,6 +1062,8 @@ spaceship_battery() {
     [[ -z $battery_data ]] && return
     battery_percent="$( echo $battery_data | awk '{print $4}' )"
     battery_status="$( echo $battery_data | awk '{print tolower($3)}' )"
+  else
+    [[ $SPACESHIP_BATTERY_ALWAYS_SHOW == true ]] || return
```

Comment:
What are you gonna show for a user, if we don't have any data about the battery? Maybe better to leave just `return`?

Reply:
Yeah I wasn't sure about this either. On one hand, there's nothing to show, on the other hand the user specifically asked to always display it.